app/src/main/java/org/openobservatory/ooniprobe/activity/runtests/adapter/RunTestsExpandableListViewAdapter.kt

```diff
@@ -0,0 +1,229 @@
+package org.openobservatory.ooniprobe.activity.runtests.adapter
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.BaseExpandableListAdapter
+import android.widget.ImageView
+import android.widget.TextView
+import org.openobservatory.ooniprobe.R
+import org.openobservatory.ooniprobe.activity.runtests.RunTestsViewModel
+import org.openobservatory.ooniprobe.activity.runtests.RunTestsViewModel.Companion.SELECT_NONE
+import org.openobservatory.ooniprobe.activity.runtests.RunTestsViewModel.Companion.SELECT_ALL
+import org.openobservatory.ooniprobe.activity.runtests.RunTestsViewModel.Companion.SELECT_SOME
+import org.openobservatory.ooniprobe.activity.runtests.models.ChildItem
+import org.openobservatory.ooniprobe.activity.runtests.models.GroupItem
+import org.openobservatory.ooniprobe.test.suite.AbstractSuite
+import org.openobservatory.ooniprobe.test.suite.ExperimentalSuite
+import org.openobservatory.ooniprobe.test.test.AbstractTest
+
+
+class RunTestsExpandableListViewAdapter(
```

Comment:
I would like to encourage you to put some documentation before each new class we introduce 🙏